com.unity.render-pipelines.core/Editor/Lighting/SRPLensFlareElementEditor.cs

```diff
@@ -0,0 +1,490 @@
+
+            // Distortion
+            SerializedProperty enableDistortionProp = property.FindPropertyRelative("enableRadialDistortion");
+            SerializedProperty targetSizeDistortionProp = property.FindPropertyRelative("targetSizeDistortion");
+            SerializedProperty distortionCurveProp = property.FindPropertyRelative("distortionCurve");
+            SerializedProperty distortionRelativeToCenterProp = property.FindPropertyRelative("distortionRelativeToCenter");
+
+            SRPLensFlareType flareType = (UnityEngine.SRPLensFlareType)flareTypeProp.enumValueIndex;
+            if (lensFlareProp.objectReferenceValue != null && flareType != SRPLensFlareType.Circle && flareType != SRPLensFlareType.Polygon)
+            {
+                Texture texture = lensFlareProp.objectReferenceValue as Texture;
+                float localAspectRatio = sizeXYProp.vector2Value.x / Mathf.Max(sizeXYProp.vector2Value.y, 1e-6f);
+                float imgWidth = 1.5f * 35.0f;
+                float usedAspectRatio = preserveAspectRatioProp.boolValue ? (((float)texture.width) / ((float)texture.height)) : localAspectRatio;
+                if (isFoldOpenedProp.boolValue)
+                {
+                    Rect imgRect = new Rect(m_CurrentRect.x + 0.5f * (position.width - imgWidth), m_CurrentRect.y + GUIStyle.none.lineHeight + 5.0f, imgWidth, imgWidth);
+                    EditorGUI.DrawTextureTransparent(imgRect, lensFlareProp.objectReferenceValue as Texture, ScaleMode.ScaleToFit, usedAspectRatio);
+                }
+                else
+                {
+                    float imgOffY = 0.5f * (GetPropertyHeight(property, label) - imgWidth - GUIStyle.none.lineHeight);
+                    Rect imgRect = new Rect(position.x - 35.0f + 15.0f, position.y + imgOffY + GUIStyle.none.lineHeight, imgWidth, imgWidth);
+                    EditorGUI.DrawTextureTransparent(imgRect, lensFlareProp.objectReferenceValue as Texture, ScaleMode.ScaleToFit, usedAspectRatio);
+                }
+            }
+            Rect rect = m_CurrentRect;
+            if (isFoldOpenedProp.boolValue && flareType != SRPLensFlareType.Circle && flareType != SRPLensFlareType.Polygon)
+            {
+                m_CurrentRect.y += 1.5f * 35.0f;
+            }
+            EditorGUI.BeginProperty(new Rect(rect.x, rect.y, rect.width, 2.0f * rect.height), label, property);
+
+            float lineHeight = EditorGUIUtility.singleLineHeight + EditorGUIUtility.standardVerticalSpacing;
+
+            Color tmpCol;
+            bool tmpBool;
+            float tmp;
+            int iTmp;
+            Vector2 tmpVec2;
+            if (EditorGUI.BeginFoldoutHeaderGroup(new Rect(position.x, position.y, position.width, GUIStyle.none.lineHeight), isFoldOpenedProp.boolValue, EditorGUIUtility.TrTextContent("Lens Flare Element")))
+            {
+                rect = GetNextRect();
+                EditorGUI.TextArea(rect, "Type", EditorStyles.boldLabel);
```

Comment:
label should be in a style class to not search them in the logic and it should be localizable.
Please use either **L18n** or **EditorGUIUtility.TrTextContent**